src/devices/ikea.ts

```diff
@@ -931,6 +932,13 @@ const definitions: DefinitionWithExtend[] = [
             battery(),
             ikeaOta(),
         ],
+        configure: async (device) => {
```

Comment:
This can probably extracted into a modernExtend quirk and added for these 2 devices.

e.g. https://github.com/Koenkk/zigbee-herdsman-converters/blob/7e8a807a334c4f21a59ef8770ceee935a9fa4c3a/src/lib/modernExtend.ts#L2350-L2359 is quirk that also adds an addition configure step.

Reply:
I think it would be better to add this to the already existing `iasZoneAlarm`, @marazmarci is this something you could change?

Reply:
@Koenkk, I implemented the changes you requested. I'm no TypeScript dev, but I hope it is good and it works.

I didn't yet have time to actually test it, so I'm marking the PR as a draft in the meantime.

Reply:
Made some small updates, let me know if this work!

Reply:
I tested it, and it worked both for PARASOLL and BADRING! I saw the `zoneStatus` attribute with the correct endpoint on the `Reporting` tab.